### ISSUE TITLE
Add support for $warn and $debug

### ISSUE THREAD

**Issue by shawnwildermuth** [anaisbetts/SassAndCoffee#45]
Would be nice if $warn and $debug would write to VS Output Window. I think:

$warn == Trace.WriteLine
$debug == Debug.WriteLine

Awesome if we could get this in the Browser's output window, but that's a significant effort, in VS's window is plenty for me.

**Comment by kogir** [comment]
If you're referring to Sass's `@warn` and `@debug` directives, I'll take a quick look and see how the Sass compiler implements them.  Can't make any promises though.

**Comment by shawnwildermuth** [comment]
Yeah, I meant @warn/@debug. I talked with Paul via twitter and he asked me to add the issue. Thanks!
